Source/DataSources/PolylineGeometryUpdater.js

```diff
@@ -626,6 +650,8 @@ define([
         geometryUpdater._clampToGround = Property.getValueOrDefault(polyline._clampToGround, time, false);
         geometryUpdater._groundGeometryOptions.positions = positions;
         geometryUpdater._groundGeometryOptions.width = Property.getValueOrDefault(polyline._width, time, 1);
+        geometryUpdater._groundGeometryOptions.lineType = Property.getValueOrDefault(polyline._lineType, time, LineType.GEODESIC);
+        geometryUpdater._groundGeometryOptions.granularity = Property.getValueOrDefault(polyline._granulatiry, time, 9999);
```

Comment:
It could be extremely expensive to always use `polyline._granularity` here, `RADIANS_PER_DEGREE` is quite small if used immediately as meters.

It'd probably be best to make `GroundPolylineGeometry` consistent with the other polyline types. The current `granularity` is there mostly just to make sure none of the segments are so long that the endpoints jitter, so we could probably switch the API `granularity` to something that gets converted to chord length but subdivide further based on `9999` under-the-hood when necessary.

The simplest way to do this for now is probably to have `GroundPolylineGeometry` accept an angular value, convert it to chord length, and `max` the value against the current default.

This won't let people produce, say, jaggedly curving rhumblines-on-terrain, but I think that's a niche enough case that it can wait either for a followup PR or until it becomes apparent.